Directory.Build.targets

```diff
@@ -6,20 +6,11 @@
     <Exclude>[GraphQL.Samples.Schemas.Chat]*,[GraphQL.Samples.Server]*</Exclude>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(OutputType)' == 'Exe'">
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
-  
   <PropertyGroup Condition="'$(IsPackable)' != 'true'">
     <NoWarn>$(NoWarn);CS1591;IDE1006;IDE0053;CS1998</NoWarn>
     <WarningsNotAsErrors>$(WarningsNotAsErrors);IDE0060</WarningsNotAsErrors>
   </PropertyGroup>
```

Comment:
This always evaluates to `true` and should not be so.  You need to add `<IsPackable>true</IsPackable>` in the `Directory.Build.props` file to fix this, along with the other changes to set `IsPackable` to `false` when appropriate.  It is too bad that msbuild does not do a better job with this, but we cannot change this.  If you find a better solution, let me know.

Reply:
>  It is too bad that msbuild does not do a better job with this

100%

> If you find a better solution, let me know.

I removed this condition and added explicit nowarns into tests.props and samples.props.